src/modclub_assets/webpack.config.cjs

```diff
@@ -167,26 +167,26 @@ module.exports = {
     new webpack.EnvironmentPlugin({
       NODE_ENV: "development",
       MODCLUB_ASSET_OLD_CANISTER_ID: "ljyte-qiaaa-aaaah-qaiva-cai",
-      CANISTER_ID_MODCLUB: canisters["modclub"] || "aaaaa-aa",
-      CANISTER_ID_MODCLUB_DEV: canisters["modclub_dev"] || "aaaaa-aa",
-      CANISTER_ID_MODCLUB_QA: canisters["modclub_qa"] || "aaaaa-aa",
-      CANISTER_ID_WALLET: canisters["wallet"] || "aaaaa-aa",
-      CANISTER_ID_WALLET_DEV: canisters["wallet_dev"] || "aaaaa-aa",
-      CANISTER_ID_WALLET_QA: canisters["wallet_qa"] || "aaaaa-aa",
-      CANISTER_ID_RS: canisters["rs"] || "aaaaa-aa",
-      CANISTER_ID_RS_DEV: canisters["rs_dev"] || "aaaaa-aa",
-      CANISTER_ID_RS_QA: canisters["rs_qa"] || "aaaaa-aa",
-      CANISTER_ID_VESTING: canisters["vesting"] || "aaaaa-aa",
-      CANISTER_ID_VESTING_DEV: canisters["vesting_dev"] || "aaaaa-aa",
-      CANISTER_ID_VESTING_QA: canisters["vesting_qa"] || "aaaaa-aa",
-      CANISTER_ID_AIRDROP: canisters["airdrop"] || "aaaaa-aa",
-      CANISTER_ID_AIRDROP_DEV: canisters["airdrop_dev"] || "aaaaa-aa",
-      CANISTER_ID_AIRDROP_QA: canisters["airdrop_qa"] || "aaaaa-aa",
-      CANISTER_ID_MODCLUB_ASSETS: canisters["modclub_assets"] || "aaaaa-aa",
+      MODCLUB_CANISTER_ID: canisters['modclub'] && canisters['modclub'][network] || "aaaaa-aa",
```

Comment:
that was made because of specific canister_ids.json  file structure, like :
  "internet_identity": { <---Canister name
    "local"( <-- Network ): "ajuq4-ruaaa-aaaaa-qaaga-cai"
  },
  "modclub_qa": {
    "local": "c2lt4-zmaaa-aaaaa-qaaiq-cai"
  },

Reply:
btw, line 36 has implemented this logic. The structure of canister_ids.json has not changed for a long time. don't see a reason why this PR would break the logic.
If needed, this change should be in a separate PR.